src/components/NavigationSide/NavigationSide.jsx

```diff
@@ -0,0 +1,26 @@
+import { NavLink } from "react-router-dom";
+import { getActiveStyleSide } from "utilities";
+import {
+	ExploreOutlined,
+	VideoLibraryOutlined,
+	FavoriteBorderOutlined,
+	WatchLaterOutlined,
+	HistoryOutlined,
+} from "@mui/icons-material";
+import "./navigation-side.css";
+
+const NavigationSide = () => {
+  return (
+    <div className="sidebar" id="video-sidebar">
+      <div className="sidebar-links mt-8">
+        <NavLink to="/explore" style={getActiveStyleSide}><ExploreOutlined />Explore</NavLink>
+        <NavLink to="/" style={getActiveStyleSide}><VideoLibraryOutlined />Playlist</NavLink>
+        <NavLink to="/" style={getActiveStyleSide}><FavoriteBorderOutlined />Liked</NavLink>
```

Comment:
Wow ! really liked the way you used function here, 
Inline style doesn't look inline anymore...

Reply:
Tanay taught this in class! It's a great way isn't it